### PR TITLE
Insert missing upgrade steps

### PR DESCRIPTION
**Pull request by rhafer**

### CODE REVIEW

File: crowbar_framework/app/controllers/installer/upgrades_controller.rb [hunk 1]
```diff
@@ -156,6 +156,9 @@ def backup
     def nodes
       @current_step = 8
       if request.post?
+        @service_object = CrowbarService.new(Rails.logger)
```
Comment (jsuchome) [inline crowbar_framework/app/controllers/installer/upgrades_controller.rb]: Actually I think you only need to adapt `before_filter :set_service_object` at the beginning

**Comment by rhafer** [comment]
@rsalevsky This are the missing step to trigger the OS upgrade, it's by no means complete, lacks error handling and shouldn't be merged (we just needed something simple for testing). You might want to use it as a base of implementing the rest of step (updating the counters and switch to the last page after the nodes are ready).

File: crowbar_framework/app/controllers/installer/upgrades_controller.rb (hunk 1 above)
Comment (rhafer) [inline crowbar_framework/app/controllers/installer/upgrades_controller.rb]: Ah, thanks for the hint. I'll update.

**Comment by jsuchome** [comment]
btw, I'd be in favor of merging it anyway, in case anyone will be testing the upgrade workflow...

**Comment by rhafer** [comment]
@MaximilianMeister  any opinion?

**Comment by MaximilianMeister** [comment]
@rhafer do we have a card for that in trello? @rsalevsky is not here until next monday.

**Comment by rhafer** [comment]
@MaximilianMeister https://trello.com/c/Z7q97UwY

**Comment by rsalevsky** [comment]
@rhafer I prefer to put both commands in a `begin ... rescue` block. Like i did it here: [upgrades_controller.rb#L112-L123](https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/app/controllers/installer/upgrades_controller.rb#L112-L123)

**Comment by rhafer** [comment]
@rsalevsky Done. Do you think we should merge this and you add the counter stuff in a later PR?

**Comment by rsalevsky** [comment]
@rhafer Definitely yes. +1

**Comment by jsuchome** [comment]
+1

**Comment by aplanas** [comment]
+1

**Pull request merged**